main/HSSF/UserModel/HSSFCell.cs

```diff
@@ -877,17 +877,41 @@ public DateTime DateCellValue
                         "You cannot get a date value from an error cell");
                 }
                 double value = this.NumericCellValue;
+                return DateUtil.GetJavaDate(value, book.IsDate1904());
+            }
+        }
+#if NET6_0_OR_GREATER
+        public DateOnly? DateOnlyCellValue 
+        { 
+            get{
+                if (CellType == CellType.Blank||CellType == CellType.String||CellType == CellType.Boolean||CellType == CellType.Error)
```

Comment:
would it be easier to check the inverse like `!= Numeric && != Formula`?

Reply:
Good suggestion